### DISCUSSION TITLE
How To Get The Type of a Model?

### DISCUSSION THREAD

**Discussion by gangradem** [prisma/prisma#18245] [Q&A] [answered]
Let's say I run this code:

```
const todos = await prisma.todo.findMany();
```

Now, this variable will be passed to other Components in my code.

What type do I define for this variable?

For example:

```
interface TaskProps {
  task: Prisma.TodoModel;
}
```

I'm new to Typescript, and I don't know how to do this. I can't find a type for the generated model...

**Comment by gangradem** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/84750225/223414279-923fc3b1-f6c6-4a62-a49b-4eb0a7960a30.png)

I can see that this Todo type is exported. But when I use Primsa.Todo as a type, it doesn't work...

**EDIT: I GOT IT NOW!**

I have to import it directly like this:

import { Todo } from "@prisma/client";

**Reply by samuel-morrissey** [reply]
Thank you brooo!
